src/main/java/www/purple/mixxy/models/User.java

```diff
@@ -26,6 +26,24 @@
 	// @Chris - TODO: how are we going to use this map?
 	// public Map<OAuthService, String> authentications = new HashMap<>();
 
+	
+	//TODO: Partial Indexing of MOD roles
+	// https://github.com/objectify/objectify/wiki/Queries#custom-conditions
+	
+//	static class IfMod extends ValueIf<User> {
+//	    public boolean matches(User user) {
+//	        return user.role == Role.MODERATOR;
+//	    }
+//
+//		@Override
+//		public boolean matchesValue(User arg0) {
+//			// TODO Auto-generated method stub
+//			return false;
+//		}
+//	}
+	
+	//TODO: Partial Indexing of MOD roles
```

Comment:
Try just [`IfNotDefault`](https://github.com/objectify/objectify/wiki/Queries#ifdefaultclass) and let the default be `USER`